src/builtins/functions.md

```diff
@@ -0,0 +1,3 @@
+#### Built-in Functions
```

Comment:
:100: on breaking apart the built-in functions based on their logical groupings. SO much more readable!